sys/include/net/nanocoap.h

```diff
@@ -1877,7 +1877,7 @@ ssize_t coap_reply_simple(coap_pkt_t *pkt,
                           unsigned code,
                           uint8_t *buf, size_t len,
                           unsigned ct,
-                          const uint8_t *payload, uint8_t payload_len);
+                          const void *payload, uint8_t payload_len);
```

Comment:
why is `payload_len` limited to `uint8_t`?

Reply:
Good question!
Looks like a copy & paste error